docs/building-on-flow/nodes/node-operation/index.md

```diff
@@ -20,15 +20,32 @@ Flow multirole architecture makes it more scalable and provides several node typ
 
 The different types of nodes are described [here](./node-roles.mdx). As node operator, you can choose to run any of the different types of node that best fits your needs.
 
-### Observer node
-The observer node is one of the easiest node to spin up and can be run by Dapp developers who need the latest block data available locally e.g. a wallet application that needs to track the latest block ID and height.
+## Light client a.k.a. Observer node
+
+The light client is one of the easiest node to spin up and can be run by Dapp developers who need the latest block data available locally e.g. a wallet application that needs to track the latest block ID and height.
 In addition to supporting dapps, an observer node can also be run by access node operators who want to scale their access nodes' endpoints. Access node operators can spin up geographically dispersed observer nodes which can talk to their staked access nodes and to each other.
 
 The observer node is not staked but still provides the same API as the access node.
 
-To run an observer node, follow this [guide](./observer-node.mdx).
+To run a light client, follow this [guide](./observer-node.mdx).
+
+## Archive node
+
+The Archive node provides a scalable and efficient way to access the history of Flow protocol and the execution state for the current spork. Like the observer node, it too can be run by anyone without being staked or added to the approved list of nodes.
+The Archive node follows the chain, stores and indexes both protocol and execution state, and allows retrieval of blocks, collections, transactions and events from the genesis of the current spork.
+
+It also allows script execution and other read-only queries that require the execution state to be read. It can answer any queries from past data e.g. “what was the Flow account balance at height X?”, where X is several thousand blocks in the past.
+
+The archive node is currently in beta and will be available as a GA release in H2 2023 (see [here](https://flow.com/post/flow-blockchain-node-operation-archive-node) for more)
 
-### Access node
+## Full node
+
+A Full node is staked node that serves the network by providing data on request (i.e. gets and shares data on the p2p network).
```

Comment:
Reading this, I feel there is a strong focus on full nodes sharing data. While that is true for some nodes (like ENs and in the future VNs) there are also nodes that don't talk to anyone other than staked nodes. 

I would suggest to differentiate full nodes vs light nodes, whether or not they contribute to protocol progress. I think for Collector, Consensus, Execution, and verification nodes that is pretty obvious. Nevertheless, also Access nodes contribute by routing transactions to collectors -- otherwise, the network could not ingest any transaction, which would make it practically useless. Furthermore, ANs help distribute data to the external world.

In comparison, the network could make meaningful progress without any other nodes. The terminology of light node continues to make sense in this context, because the nodes does not have to contribute any resources to network progress. In comparison, Full nodes are designed to _additionally_ provide resources for protocol-internal tasks. 

ANs are a bit of a hybrid between both worlds. Nevertheless, we indent for node operators to at least support the network with some minimal contributions, like routing transactions from other people, very rate-limited script execution, helping to distribute data to the unstaked P2P network. While an operator can run their AN solely for themselves and not provide any services to the community, it is certainly not the intended way of operating an AN. Hence, I feel the classification of an AN as full node is warranted (as you have it in your diagram).


```suggestion
In a nutshell, Full Nodes are staked network participants that drive network progress, e.g. by creating and executing new block. They are the primary contributors to network safety (all of them validate the correctness of the consensus process and secure the network additionally through their role-specific tasks). In comparison, Light Nodes don't contribute to the networks progress. Though, they help to secure the network by also validating the integrity of the consensus process. 
```

Reply:
thank you - committed and reworded a lil.